byline.go

```diff
@@ -145,17 +145,15 @@ func (lr *Reader) MapStringErr(filterFn func(string) (string, error)) *Reader {
 	})
 }
 
-// Each - processing each line without changing the line
+// Each - processing each line
```

Comment:
Under the hood is a slice that returns `scanner.Bytes()`.
See https://github.com/golang/go/blob/master/src/bufio/scan.go#L139 for implementation. This is a bit complicated code :)